src/core/utilities.js

```diff
@@ -94,10 +94,14 @@ class Utilities {
                 default:
                     break;
             }
-
+            console.log("Sending Message: ");
+            console.log(messageJson);
```

Comment:
Change it to ` console.log("Sending Message: ", messageJson);`

Reply:
done, please chek